bibliopixel/project/aliases.py

```diff
@@ -0,0 +1,56 @@
+import copy
+from .importer import import_symbol
+
+ALIASES = {
+    'driver': {
+        'apa102': 'bibliopixel.driver.APA102.DriverAPA102',
+        'dummy': 'bibliopixel.drivers.dummy_driver.DriverDummy',
+        'hue': 'bibliopixel.driver.hue.DriverHue',
+        'image': 'bibliopixel.driver.image_sequence.DriverImageSequence',
+        'lpd8806': 'bibliopixel.driver.LPD8806.DriverLPD8806',
+        'network': 'bibliopixel.driver.network.DriverNetwork',
+        'network_udp': 'bibliopixel.driver.network.DriverNetworkUDP',
+        'simpixel': 'bibliopixel.driver.SimPixel.DriverSimPixel',
+        'ws2801': 'bibliopixel.driver.WS2801.DriverWS2801',
+    },
+
+    'led': {
+        'circle': 'bibliopixel.led.circle.LEDCircle',
+        'cube': 'bibliopixel.led.cube.LEDCube',
+        'matrix': 'bibliopixel.led.matrix.LEDMatrix',
+        'pov': 'bibliopixel.led.pov.LEDPOV',
+        'strip': 'bibliopixel.led.strip.LEDStrip',
+    },
+
+    'animation': {
```

Comment:
I love all of this except this since this list will either be completely arbitrary to what you and I happen to need at the time and therefore incomplete or we'd have to maintain it so that it has ALL animations in BPA. Which isn't going to happen. Personally I'd keep it to only the stuff under `bibliopixel.animations` and leave out the BPA stuff since those don't ever have multiple ways of importing them. They are *always* BPA.<type>.<something>